REBASE.openshift.md

```
@@ -58,6 +58,46 @@ number of the OCPBUGS ticket created in step (1).
+
+When rebasing openshift/etcd to a new upstream etcd that bumps the
+bbolt version for a new release:
+
+1. Create a new `release-X.Y` branch on `openshift/bbolt` from the new
+   bbolt tag (e.g. `release-5.1` from `v1.4.4`)
+2. Cherry-pick the MADV_RANDOM removal patch onto it
+3. Update the replace directive in all four `go.mod` files
+   (root, server, etcdutl, tests). Use `go mod edit` with the raw
+   commit hash and then `go mod tidy` to resolve the correct
+   pseudo-version (timestamp + hash), e.g.:
+   ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)